docs/src/code-examples/sending-emails/java.mdx

```diff
@@ -3,34 +3,31 @@ lang: java
 ---
 
 
+`client.sendEmail` is an overloaded method with multiple ways to call it. You can supply an `emailReplyToId` if it suits your system, or leave it off.
```

Comment:
Could probably do without this comment comment and just have the method signature